daemon/algod/api/algod.oas2.json

```diff
@@ -1150,6 +1150,65 @@
         }
       }
     },
+    "/v2/transactions/simulate": {
+      "post": {
+        "consumes": [
+          "application/x-binary"
+        ],
+        "produces": [
+          "application/msgpack"
+        ],
+        "schemes": [
+          "http"
+        ],
+        "summary": "Simulates a raw transaction as it would be evaluated on the network.",
```

Comment:
I realize this is probably inheriting bad docs from the raw transaction endpoint, but let's be clearer here. The endpoint accepts a group, right?

Reply:
Good point, I've clarified in 3f4e74e9059205c7785d9738911d2760169ae4f4, and I also took the opportunity to update the raw transaction endpoint.